build.gradle

```diff
@@ -91,13 +91,13 @@ repositories {
 
 dependencies {
 	compile 'com.dell.isg.smi:wsmanlib:1.0.29'
-	compile 'com.dell.isg.smi:wsmanclient:1.0.28'
+	compile 'com.dell.isg.smi:wsmanclient:1.0.17'
     compile 'com.dell.isg.smi:racadmlib:1.0.16'
-  	compile 'com.dell.isg.smi:commons-elm:1.0.76'
-	compile 'com.dell.isg.smi:commons-model:1.0.81'
+  	compile 'com.dell.isg.smi:commons-elm:1.0.33'
+	//compile 'com.dell.isg.smi:commons-model:1.0.65'
```

Comment:
Model should be version 80.  http://repo1.maven.org/maven2/com/dell/isg/smi/commons-model/1.0.80/